Sources/Reactive.swift

```diff
@@ -0,0 +1,26 @@
+/// Describes a class which has been extended with reactive elements.
+///
+/// - note: `ExtendedForReactiveness` is only intended for extensions to types
+///         that are not owned by the module. Non-conforming types may carry
+///         first-party reactive elements.
+public protocol ExtendedForReactiveness: class {}
```

Comment:
How about `ReactiveType` for this?

I know it's a bit outside of the Swift norm, but I think it could work here.

1. `Reactive` is the obvious choice, but the `struct` is `Reactive`. (Which I absolutely think is the right name.)

2. `ReactiveProtocol` might be useful to apply to `Reactive` itself

3. `ReactiveType` is meant to suggest that the type itself is reactive.